### PR TITLE
General features for portfolio

### PR DESCRIPTION
**Pull request by lexoxo**
This branch resume all the work made for the project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,46 @@
...
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (medaminedev66) [inline README.md:46]: - [ ] The MIT link is broken. Kindly fix this. Your README is the entry point of your users and having it well written without any broken links gives a good first impression and maximizes the potential of your product to your users.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    <h1>Hey there.I'am Axel.<br><span id="second-h1">i'm a software developer</span></h1>
+    <p>I can help you build your product, feature or website look through some of my work and<br> axperience! 
+       If you like what you see and have a project you need coded,<br> don't hesitate to contact me</p>
+       <div class="social">
+        <i class="fa fa-github"></i>
+        <i class="fa fa-linkedin"></i>
+        <i class="fa fa-hand-peace-o"></i>
+        <i class="fa fa-twitter"></i>
+        <i class="fa fa-envelope-o"></i>
+       </div>
```
[the remark this answers is not in the document]
Comment (lexoxo) [reply index.html]: Thanks i corrected it

File: README.md (hunk 1 above)
Comment (lexoxo) [reply README.md:46]: Thanks for the feedback, i corrected it

File: index.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+       If you like what you see and have a project you need coded,<br> don't hesitate to contact me</p>
+       <ul class="social">
+        <li><i class="fa fa-twitter"></i></li>
+        <li><i class="fa fa-linkedin"></i></li>
+        <li><i class="fa fa-envelope-o"></i></li>
+        <li><i class="fa fa-github"></i></li>
+        <li><i class="fa fa-hand-peace-o"></i></li>
+        </ul>
+       <img class="more-img" src="images/more.png" alt=""/>
+</section>
```
Comment (gemmen29) [inline index.html:32]: - Please change the font color and weight to match the Figma design.
- Kindly adjust the width of the paragraph as it's touching the edge of the screen.
- Please change the size of the social media icons to match the Figma design and remove the dots before each list item using `list-style: none`.

| Your design | Figma design |
| --- | --- |
| image | image |

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @lexoxo, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (lexoxo) [reply index.html]: Thanks you for the feedback. I resolved the problems

Comment (lexoxo) [reply index.html]: I added some padding to fix edges problems

File: index.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+       <ul class="social">
+        <li><i class="fa fa-twitter"></i></li>
+        <li><i class="fa fa-linkedin"></i></li>
+        <li><i class="fa fa-envelope"></i></li>
+        <li><i class="fa fa-github"></i></li>
+        <li><i class="fa fa-hand-peace-o"></i></li>
+        </ul>
+       <img class="more-img" src="images/more.png" alt=""/>
+      
+</section>
```
Comment (mateo951) [inline index.html:31]: Addressed 🙌 The design is looking really good 👌

**Review by mateo951 — APPROVED** [review]
### Hi @lexoxo, 

Fantastic job implementing all the changes required by your previous reviewers 🔥
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**